walnut/converter/converter.go

```diff
@@ -127,32 +130,38 @@ func ExtractTaskClasses(dplDump Dump, envModules []string) (tasks []*task.Class,
 			}
 		}
 
-		// fmt.Printf("\nTASK:\n%v\n", task)
 		tasks = append(tasks, &task)
 	}
 	return tasks, nil
 }
 
-// GenerateTaskTemplate takes as input an array of pointers to task.Class
-// and writes them to a AliECS friendly YAML file
-func GenerateTaskTemplate(extractedTasks []*task.Class) (err error) {
-
-	// Check if "tasks" directory exists. If not, create it
-	_, err = os.Stat("tasks")
-	if os.IsNotExist(err) {
-		// FIXME: use absolute paths
-		errDir := os.Mkdir("tasks", 0755)
-		if errDir != nil {
-			return fmt.Errorf("create dir failed: %v", err)
+func sanitizeCmdLineArgs (input []string, taskName string) (output []string) {
+	for _, value := range input {
+		// Check args for dump arguments and remove them
+		if  strings.Contains(value, "--dump-workflow") ||
+			strings.Contains(value, "--dump-workflow-file") {} else {
+				output = append(output, value)
```

Comment:
Also remove the entry immediately after `--dump-workflow-file` if said entry is `*.json` as previously mentioned.